adobe/react-spectrum
# getTimeZoneOffset: fix offset bug for old dates (#3924)

Fixes issue where `getTimeZoneOffset` would not include seconds offset (applicable for old dates) when system time zone equalled passed time zone.

Closes #3924 

## ✅ Pull Request Checklist:

   Not sure how to unit test this one – it would have to change system `TZ`
- [ ] ~Updated documentation (if it already exists for this component).~ (Not applicable)
- [ ] ~Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)~ (Not applicable)

## 📝 Test Instructions:

- Set browser timezone to `Europe/Oslo`
- Use [this codebox example]( https://codesandbox.io/s/happy-wozniak-838vce)
- See the bug
- Update code as in this PR (Search for `Fast path: for local timezone` in `node_modules/@internationalized`)
- See bug not appearing (anymore)
